eq-author/src/App/page/Logic/Routing/RoutingPage/RoutingEditor/RuleEditor/index.js

```diff
@@ -116,12 +116,27 @@ export class UnwrappedRuleEditor extends React.Component {
 
     const existingRadioConditions = {};
 
-    const validationErrorInfo = this.props.rule.validationErrorInfo;
+    const validationErrorInfo = rule.validationErrorInfo;
 
     const validationErrors = validationErrorInfo.totalCount
       ? validationErrorInfo.errors
       : [];
 
+    const matchSelectErrors = expressions.filter(({ validationErrorInfo }) => {
+      if (!validationErrorInfo || !validationErrorInfo.totalCount) {
+        return false;
+      }
+      const expressionGroupOperatorErrors = validationErrorInfo.errors.filter(
+        ({ field }) => field === "groupOperator"
+      );
+
+      if (expressionGroupOperatorErrors.length) {
+        return true;
+      }
+
+      return false;
```

Comment:
This is also causing the coverage to drop

Reply:
I've put in a test for this.